vendor/github.com/openshift/api/security/v1/generated.pb.go

```diff
@@ -846,6 +846,30 @@ func (m *SecurityContextConstraints) MarshalTo(dAtA []byte) (int, error) {
 			i += n
 		}
 	}
+	if m.DefaultAllowPrivilegeEscalation != nil {
```

Comment:
This needs to be done in the openshift/api repo and then backported here.  I do not think a "carry" in a repo we control is appropriate.